mixlib-shellout.gemspec

```diff
@@ -13,6 +13,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 2.2"
 
+  s.add_dependency "chef-utils"
```

Comment:
this is deliberately without a constraint, since this would introduce cross-repo constraints between this git repo and the chef/chef git repo.

IMO this is screaming out for monorepo and moving this into chef/chef.

but alternatively the pins can live in the project and we can have a relaxed requirement here.  but this means that consumers are responsible for knowing which version(s) work (which really in our kinda-CI/CD world are "the latest works with the latest").

today for example, this pin would probably be >= 16.4.0, but we still haven't shipped chef-utils-16.3.x yet.